app/src/components/dialog/YesNoDialog.tsx

```diff
@@ -73,18 +73,22 @@ export interface IYesNoDialogProps {
   /**
    * Optional yes-button props
    *
-   * @type {Partial<ButtonProps>}
-   * @memberof IYesNoDialogProps
+   * @type {any}
+   * Needed fix: Add correct hardcoded type.
+   * Note: LoadingButtonProps causes build compile issue
+   * https://github.com/mui/material-ui/issues/30038
    */
-  yesButtonProps?: Partial<LoadingButtonProps>;
+  yesButtonProps?: any;
```

Comment:
Does the `ButtonProps` type work? If so, we could use `type LoadingButtonProps = ButtonProps & { loading?: boolean }`

Reply:
Per discussion, this change introduces a large performance bottleneck.